sdk/rust-examples/huffman-encoding/README.md

```diff
@@ -0,0 +1,41 @@
+# A simple huffman encoding implementation with Veracruz freestanding execution engine
+
+
+This is a simple program that lets you implement the "Huffman Encoding Algorithm" which is a Lossless Compression Algorithm used to compresss Data. It was developed by David A. Huffman.  
+This directory is a simple program that takes a file called `huffman_example_input.txt` and compresses its data using the Huffman Encoding Algorithm, and then writes teh output into `encoded_output.txt`
+
+# Steps
+
+1. Go to `sdk/freestanding-execution-engine` directory.
+
+2. run `cp -r ../rust-examples/huffman-encoding .`. This will copy the current directory to the Veracruz freestanding execution engine.
+
+3. Inside `sdk/freestanding-execution-engine`, Create a directory called `input`, and another directory called `output`.
+
+4. Run the following command :
+```
+cp ../../test-collateral/huffman_example_input.txt input
+```
+This will copy the "Veracruz README file" to the input directory. This will be our input to the program.
+
+5. Go to `huffman-encoding` directory and run the following commands :

```

Comment:
These commands should be in a Makefile. You can have a look at the other examples

Reply:
I will take a look at the other examples.
Thank you for this valuable suggestion!!